pandas/tests/test_categorical.py

```diff
@@ -1481,12 +1481,12 @@ def test_repr(self):
 
         self.assertEqual(exp, a.__unicode__())
 
-        a = pd.Series(pd.Categorical(["a","b"] *25, name="a"))
-        exp = u("".join(["%s    a\n%s    b\n"%(i,i+1) for i in range(0,10,2)]) + "...\n" +
-                "".join(["%s    a\n%s    b\n"%(i,i+1) for i in range(40,50,2)]) +
-                "Name: a, Length: 50, dtype: category\n" +
-                "Categories (2, object): [a < b]")
-        self.assertEqual(exp,a._tidy_repr())
+        # a = pd.Series(pd.Categorical(["a","b"] *25, name="a"))
+        # exp = u("".join(["%s    a\n%s    b\n"%(i,i+1) for i in range(0,10,2)]) + "...\n" +
+        #         "".join(["%s    a\n%s    b\n"%(i,i+1) for i in range(40,50,2)]) +
+        #         "Name: a, Length: 50, dtype: category\n" +
```

Comment:
hmm, can't take this out. assume the cat repr needs fixing? or is this just a dupe test